apps/portal/src/app/globals.css

```diff
@@ -93,6 +93,15 @@ html {
   }
 }
 
+/* If no data-theme is added in body, its in dark theme  */
+body:not([data-theme="light"]) .light-only {
+  display: none;
+}
+
+[data-theme="light"] .dark-only {
+  display: none;
+}
+
```

Comment:
is this due to chakra & tailwind mixing?

Reply:
No, we don't have chakra in portal, We have custom "theme provider" setup that sets a data-theme attribute in portal